### DISCUSSION TITLE
Logo returns to default after reboot

### DISCUSSION THREAD

**Discussion by jiyuuule** [fastfetch-cli/fastfetch#598] [Q&A]
I used "fastfech --logo" command
after rebooting my PC or terminal logo returns to default 
![Screenshot_20231028_183722](https://github.com/fastfetch-cli/fastfetch/assets/127411012/c0a83bdb-9fc9-4922-9b02-2f127aae64fc)
![Screenshot_20231028_183909](https://github.com/fastfetch-cli/fastfetch/assets/127411012/e5ed40be-a9fb-4f96-a167-4d83ac4d29e2)

**Comment by CarterLi** [comment]
Command line switches are temporary. You need to put it into the config file.

**Reply by jiyuuule** [reply]
Thank you
Unfortunately I have a new problem.
Everything works 1-2 times and then only information without a logo is displayed. This happens in the standard terminal and Kitty. Manually everything is fine :\

![Screenshot_20231028_201056](https://github.com/fastfetch-cli/fastfetch/assets/127411012/1e0f0b88-9843-4851-b480-677f0b021f30)

![Screenshot_20231028_201136](https://github.com/fastfetch-cli/fastfetch/assets/127411012/2584be79-5664-4bad-8bf2-056d0de33972)

**Comment by CarterLi** [comment]
See https://github.com/fastfetch-cli/fastfetch/blob/dev/.github/ISSUE_TEMPLATE/bug_report.md?plain=1#L71
